Source/CISQLite3/CISQLite3.Build.cs

```diff
@@ -11,9 +11,9 @@ public CISQLite3(ReadOnlyTargetRules Target) : base(Target)
     PublicIncludePaths.AddRange(new string[] { Path.Combine(ModuleDirectory, "Public") });
     PrivateIncludePaths.AddRange(new string[] {  Path.Combine(ModuleDirectory, "Private") });
 
-    PrivatePCHHeaderFile = "Private/CISQLite3PrivatePCH.h";
-
-    PublicDependencyModuleNames.AddRange(
+    PrivatePCHHeaderFile = "Private/CISQLite3PrivatePCH.h";
+
+        PublicDependencyModuleNames.AddRange(
```

Comment:
Not sure why this indentation happened, this change seems to be whitespace only?